stylesheets/_component.panels.scss

```diff
@@ -136,3 +136,7 @@
     margin: 10px 0 0;
     text-align: right;
 }
+
+.panel--danger a.view-history-link {
```

Comment:
Too specific, panels should already set link colours, need to investigate why this isn't happening  

https://github.com/jadu/pulsar/blob/be350a2ce8322122c97d38100c81dda7d82c9a4a/stylesheets/_component.panels.scss#L81